src/server.c

```diff
@@ -4575,6 +4576,17 @@ int finishShutdown(void) {
                   num_replicas);
     }
 
+    if (safe && server.cluster_enabled && clusterNodeIsVotingPrimary(getMyClusterNode())) {
+        if (force) {
+            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to down. Exit anyway.");
+        } else {
+            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to down, can't exit.");
```

Comment:
```suggestion
            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to go down, can't exit.");
```

Reply:
@zuiderkwast Do you have any suggestions for texts? I usually feel like my English texts are very Chinese in sometimes.

Reply:
@jsoref's suggestion is good. You can take it.

Reply:
* I am → This is
* to down → to go down
* , → .


```suggestion
            serverLog(LL_WARNING, "This is a voting primary. Shutting down may cause the cluster to go down. Can't exit.");
```